html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <label for="email">Email</label>
+    <br />
+    <input type="text" id="email" name="name" required minlength="4" maxlength="18" size="10">
+    <br />
+    <label for="options">Why is coding cool?</label>
+    <br />
+    <select name="Select Options">
+      <option value="option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+
+
+    <p>Favorite Language (choose one)</p>
+    <input type="radio" name="language"
```

Comment:
each of these inputs needs a label, you can wrap the input with the label if desired